polling_stations/apps/councils/management/commands/import_councils.py

```diff
@@ -1,4 +1,5 @@
 import time
+import json
 
 import requests
 from bs4 import BeautifulSoup
```

Comment:
This import is now not used. In fact, can we completely remove BS4 from the project requirements?